src/libpgagroal/prometheus.c

```diff
@@ -2021,3 +2065,164 @@ append_ullong(char* orig, unsigned long long l)
 
    return orig;
 }
+
+/**
+ * Checks if the Prometheus cache configuration setting
+ * (`metrics_cache`) has a non-zero value, that means there
+ * are seconds to cache the response.
+ *
+ * @return true if there is a cache configuration,
+ *         false if no cache is active
+ */
+static bool
+is_metrics_cache_configured(void)
+{
+   struct configuration* config;
+
+   config = (struct configuration*)shmem;
+
+   // cannot have caching if not set metrics!
+   if (config->metrics == 0)
+   {
+      return false;
+   }
+
+   if (config->metrics_cache_max_age != PGAGROAL_PROMETHEUS_CACHE_DISABLED)
+   {
+      return true;
+   }
+   else
```

Comment:
Just return false, no need for the `else`

Reply:
Fine, I was not sure about styling...